packages/protocol/contracts/L1/verifiers/PseZkVerifier.sol

```diff
@@ -29,68 +44,106 @@ contract PseZkVerifier is EssentialContract, IVerifier {
 
     /// @inheritdoc IVerifier
     function verifyProof(
-        // blockId is unused now, but can be used later when supporting
-        // different types of proofs.
-        uint64,
-        address prover,
-        bool isContesting,
-        TaikoData.BlockEvidence calldata evidence
+        TaikoData.BlockEvidence calldata evidence,
+        Input calldata input
     )
         external
         view
     {
         // Do not run proof verification to contest an existing proof
-        if (isContesting) return;
-
-        bytes32 instance = calcInstance(prover, evidence);
+        if (input.isContesting) return;
+
+        PseZkEvmProof memory proof = abi.decode(evidence.proof, (PseZkEvmProof));
+
+        bytes32 instance;
+        if (input.blobUsed) {
+            PointProof memory pf = abi.decode(proof.pointProof, (PointProof));
+
+            instance = calcInstance({
+                prover: input.prover,
+                evidence: evidence,
+                pointValue: pf.pointValue,
+                metaHash: input.metaHash
+            });
+
+            Lib4844.evaluatePoint({
+                blobHash: input.blobHash,
+                x: calc4844PointEvalX(input.blobHash, pf.txListHash),
```

Comment:
@Brechtpd please take a look at this file. 
x is calculatd based on blobHash and txListHash, bacause of this, txListHash is currently not hashed into instance. Let me know if you think this is OK?

Reply:
It needs to be part of the instance: https://github.com/taikoxyz/taiko-mono/pull/15031#discussion_r1372355040

Reply:
fixed